config/config.template.json

```diff
@@ -0,0 +1,34 @@
+{
+  "irc": {
+    "presence": "Mechasqueak3[BOT]",
```

Comment:
While we're at it, can we change this to `nickname`? Presence always sounded very ominous and confused the bananas out of me at first.

Reply:
lol ok. Methinks i used "presence" as that is what pydle called it. Will adjust